app/controllers/pages_controller.rb

```diff
@@ -11,8 +11,10 @@ def determine_layout
       'frontend'
     end
 
-    # @note for now, just grab three randomly published work versions
-    def work_versions
-      WorkVersion.where(aasm_state: 'published').order(Arel.sql('RANDOM()')).limit(3)
+    def resources
+      FeaturedResource
```

Comment:
I would do `.includes(:resource)` here to eager load the resources and eliminate the n+1 query bug. Granted, you know that your _n_ is going to be 3-5, but I feel like it's a good habit to look out for and correct these situations.

Reply:
@rschenk I've added it, but I may need more explanation as to what it's actually doing. Is it going to preload the resource (work, work version, or collection) for _every_ record in the table?